acceptance/suites/tests/authorization/default_rules.rb

```diff
@@ -58,7 +56,8 @@ def assert_denied(expected_stdout)
 def report_query(node)
   curl = "/puppet/v3/report/#{node}?environment=production "
   curl += '-X PUT -H "Content-Type: text/pson" '
-  curl += '--data "{\"host\":\"$(hostname -f)\", \"metrics\":{}, \"logs\":[], \"resource_statuses\":{}}"'
+  curl += '--data "{\"host\":\"' + node
+  curl += '\",\"metrics\":{},\"logs\":[],\"resource_statuses\":{}}"'
```

Comment:
Escaping sucks.  All I did was s/$(hostname -f)/node and then broke it into two lines just to avoid a long line